src/Ansatz/Chain.jl

```diff
@@ -547,6 +570,61 @@ function evolve_2site!(qtn::Chain, gate::Dense; threshold, maxdim)
     return qtn
 end
 
+"""
+    θ(ψ::Chain, site::Site)
+
+For a [`Chain`](@ref) in the canonical form, returns the θ tensor (ΛᵢΓᵢΛᵢ₊₁Γᵢ₊₁Λᵢ₊₂), where i is the `site` index.
+"""
+function get_θ(ψ::Chain, i::Site)
```

Comment:
Rename this function. Using `get` or `set` is considered bad practice. Also using just $\theta$ doesn't tell much info and can be confusing.

Reply:
Okey, I will not use `get`, maybe `contract_\theta` or something... I think that using $\theta$ is a good idea here since it's commonly used in the literature for the two-site wave function.